byte RECONFIGURE_KALMAN_FILTER_FLAG = 0
byte RECONFIGURE_INPUT_SIGMA = 1
byte RECONFIGURE_BIAS_SIGMA = 2
byte RECONFIGURE_MEASURE_SIGMA = 3

